tutorials/plot_metadata_epochs.py

```diff
@@ -0,0 +1,118 @@
+"""
```

Comment:
yeah this tutorial is in very rough form - it'll probably change a lot but I wanted it up here to give an idea for the API etc